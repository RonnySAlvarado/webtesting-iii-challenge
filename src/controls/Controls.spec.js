import React from "react";
import "@testing-library/react/cleanup-after-each";
import "jest-dom/extend-expect";
import Controls from "./Controls.js";
import { render } from "@testing-library/react";

it("renders without crashing", () => {
  render(<Controls />);
});
