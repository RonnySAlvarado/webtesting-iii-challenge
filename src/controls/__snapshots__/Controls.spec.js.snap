// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Controls /> should match snapshot 1`] = `
<div
  className="controls panel"
>
  <button
    className="toggle-btn"
    data-testid="toggle-lock"
    disabled={true}
  >
    Lock Gate
  </button>
  <button
    className="toggle-btn"
    data-testid="toggle-open"
  >
    Close Gate
  </button>
</div>
`;
